src/middlewared/middlewared/plugins/truecommand/__init__.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
+
+
+async def _event_system(middleware, event_type, args):
+    if args['id'] == 'ready':
+        await middleware.call('truecommand.start_truecommand_service')
+    elif args['id'] == 'shutdown':
+        # Stop wireguard here please if we have it enabled
+        await middleware.call('truecommand.stop_truecommand_service')
```

Comment:
Do we have to care about that? Also wont the service be stopped anyway because of rc.d normal shutdown process?